app/models/container_image.rb

```diff
@@ -11,6 +11,7 @@ class ContainerImage < ApplicationRecord
   DOCKER_PULLABLE_PREFIX = "docker-pullable://".freeze
   DOCKER_PREFIXES = [DOCKER_IMAGE_PREFIX, DOCKER_PULLABLE_PREFIX].freeze
 
+  belongs_to :container_image, :polymorphic => true
```

Comment:
I though we needed this line, but now I dont think so anymore. I will remove this.

https://github.com/ManageIQ/manageiq/pull/15386#discussion_r122469016